### ISSUE TITLE
Security updates for perl-base and libgcrypt20

### ISSUE THREAD

**Issue by u007** [debuerreotype/docker-debian-artifacts#41]
hi,

i think there is a security update for debian stretch,
is it updated yet?

thank you

**Comment by mt-clearhaus** [comment]
It is not updated. I am also awaiting an update :slightly_smiling_face: 

Issues:

* https://www.debian.org/security/2018/dsa-4226 (perl-base, patched in 5.24.1-3+deb9u4)
* https://www.debian.org/security/2018/dsa-4231 (libgcrypt20, patched in 1.7.6-2+deb9u3)

Status:

```
$ docker pull debian:stretch
stretch: Pulling from library/debian
Digest: sha256:de3eac83cd481c04c5d6c7344cd7327625a1d8b2540e82a8231b5675cef0ae5f
Status: Image is up to date for debian:stretch

$ docker run --rm -it debian:stretch bash -c 'dpkg -l | egrep "perl-base|libgcrypt20" | sed "s/   */ /g" | cut -d " " -f 2,3'
libgcrypt20:amd64 1.7.6-2+deb9u2
perl-base 5.24.1-3+deb9u3
```

@u007 Renaming the issue to e.g. "Security updates for perl-base and libgcrypt20" might be appropriate.

**Comment by tianon** [comment]
https://github.com/docker-library/official-images/pull/4497 :+1:
